.github/workflows/hygiene-check.yml

```
@@ -0,0 +1,173 @@
+            echo "✅ **Excellent** - Code meets high hygiene standards" >> $GITHUB_STEP_SUMMARY
+          elif [ "$HYGIENE_SCORE" -ge 80 ]; then
+            echo "✅ **Good** - Code meets hygiene standards" >> $GITHUB_STEP_SUMMARY
+          elif [ "$HYGIENE_SCORE" -ge 70 ]; then
+            echo "⚠️ **Fair** - Some hygiene improvements recommended" >> $GITHUB_STEP_SUMMARY
+          else
+            echo "❌ **Needs Improvement** - Hygiene fixes required" >> $GITHUB_STEP_SUMMARY
+          fi
+          
+      - name: Comment on PR (if applicable)
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)